FhirToDataLake/scripts/Set-SynapseEnvironment.ps1

```diff
@@ -79,6 +79,55 @@ else
     throw " -> The FHIR version '$FhirVersion' is not supported."
 }
 
+function Start-Retryable-Job {
+    [CmdletBinding()]
+    Param(
+        [string]$Name, [scriptblock]$ScriptBlock, [Object[]]$ArgumentList 
+    )
+
+    $prefixBlock = '
+        $retryCount = 0
+        $retryMax = 2
+        $delay = 300
+            
+        do {
+            try 
+            {
+    '
+    $suffixBlock = '
+                return
+            }
+            catch [Exception]
+            {
+                $retryCount++
+                if ($retryCount -le $retryMax)
```

Comment:
Seems it only retries one time when fails?

Reply:
Retry twice, the **-le** is less than or equal to.